.github/workflows/test-gh.yml

```diff
@@ -43,6 +43,8 @@ jobs:
         mkdir /tmp/bin
         export PATH=/tmp/bin:$PATH
 
+        ./hack/verify-no-dirty-files.sh
```

Comment:
this probably need to be at the end, since nothing has happened that would have resulted in dirty git.

Reply:
`verify-no-dirty-files` is running `./hack/build.sh` so we should see diff. FWIW, it's failing now in CI cause there appears to be a vendor diff: https://github.com/vmware-tanzu/carvel-kapp-controller/runs/2842176794?check_suite_focus=true

Reply:
ah i totally missed it. didnt expect that verify-* script was going to do a build based on the name, but fair enough.